JS/Lesson24/goksel/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+body {
+    background-color: #000;
+}
+
+.form-container {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+}
+
+/* .form is for to remove content */
+.form {
+    display: none;
+}
+
+/* .active-form is for to make active content of current form */
+.active-form {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+    display: block;
+}
+
+#myForm {
+    display: flex;
+}
+
+/* codes for to edit main form container*/
+.left-container {
+    padding: 35px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.left-container>h1 {
+    text-align: center;
+    font-weight: 800;
+    font-size: 44px;
+}
+
+.left-container>ul {
+    margin-bottom: 45px;
+}
+
+.left-container>ul>li {
+    list-style-type: none;
+    font-weight: 300;
+    margin-left: -45;
```

Comment:
Negative margins are never good. Remove padding-left from the form element. You'll see it if you inspect the form element in the browser.